shopfloor/services/checkout.py

```diff
@@ -592,7 +595,9 @@ def _switch_line_qty_done(self, picking, selected_lines, switch_lines):
 
     @staticmethod
     def _filter_lines_unpacked(move_line):
-        return move_line.qty_done == 0 and not move_line.shopfloor_checkout_done
+        return (
+            move_line.qty_done == 0 or move_line.shopfloor_user_id == move_line.env.user
```

Comment:
May be with this ?

move_line.qty_done == 0 or move_line.shopfloor_user_id

Reply:
agree